envs/monkey_zoo/blackbox/test_blackbox.py

```diff
@@ -521,9 +522,33 @@ def test_depth_2_a(self, island_client):
         )
 
     def test_depth_1_a(self, island_client):
-        TestMonkeyBlackbox.run_exploitation_test(
-            island_client, depth_1_a_test_configuration, "Depth1A test suite", masque=b"m0nk3y"
+        test_name = "Depth1A test suite"
+        masque = b"m0nk3y"
+
+        expected_credentials: List[str] = [
+            "fc525c9683e8fe067095ba2ddc971889",
+            "e1c0dc690821c13b10a41dccfc72e43a",
+            "vakaris_zilius",
+            "m0nk3y",
+        ]
+
+        mimikatz_analyzer = MimikatzAnalyzer(island_client, expected_credentials)
```

Comment:
This passes a `List[str]` for `expected_credentials` to `MimikatzAnalyzer`, but `MimikatzAnalyzer` expects `List[Credentials]`

If possible, we should pass in a `List[Credentials]`

Reply:
The trouble is that the tests are still ambiguous. Of The expected stolen credentials, what is a username? What is a password? Are there any passwords, or are they hashes? LM or NTLM?